.gitlab-ci.yml

```diff
@@ -0,0 +1,45 @@
+include:
+  - project: 'city-of-helsinki/kuva/ci-cd-config/ci-configuration'
+    ref: v2
+    file: '/.gitlab-ci-template.yml'
+
+variables:
+  SERVICE_PORT: 80
+
+build-review:
+  # These variables are available only for review env and are merged with the general variables defined above.
+  extends: .build
+  variables:
+    DOCKER_IMAGE_NAME: '$CI_PROJECT_NAME-review'
+  only:
+    refs:
+      - external_pull_requests
+
+build-staging:
+  extends: .build
+  variables:
+    DOCKER_IMAGE_NAME: '$CI_PROJECT_NAME-staging'
+  only:
+    refs:
+      - develop
+
+build-production:
+  extends: .build
+  variables:
+    DOCKER_IMAGE_NAME: '$CI_PROJECT_NAME-production'
+  only:
+    refs:
+      - master
```

Comment:
There is no need to define multiple builds if all the build jobs end up building the same image (i.e. no build-time arguments are passed to `docker build`).

Reply:
Ok! But actually I will be adding a couple of `DOCKER_BUILD_ARG_*`s to those in the next PR I create, so if those multiple builds are then needed anyway, I think I don't bother to change that back and forth :)